mungegithub/github/github.go

```diff
@@ -726,40 +747,45 @@ func (obj *MungeObject) RemoveLabel(label string) error {
 // GetHeadAndBase returns the head SHA and the base ref, so that you can get
 // the base's sha in a second step. Purpose: if head and base SHA are the same
 // across two merge attempts, we don't need to rerun tests.
-func (obj *MungeObject) GetHeadAndBase() (headSHA, baseRef string, ok bool) {
+//CHECK-UPDATED
+func (obj *MungeObject) GetHeadAndBase() (headSHA, baseRef string, ok bool, err error) {
```

Comment:
Revert this change. I believe that `ok` is supposed to indicate if there was an error. Maybe we should use that pattern more often than actually returning the error. What are your thoughts?